.github/issue_template.md

```diff
@@ -1,14 +1,10 @@
-## What Needs to be Documented?
-
-
-## Why Should This Change Be Made?
+<!--
+Thanks for wanting to contribute to the ownCloud documentation!
 
+However, before reporting any issues, please make sure that you're using the latest available version of master.
```

Comment:
`latest available version of master.`--> `latest available documentation version of master.`